Dockerfile

```
@@ -0,0 +1,14 @@
+FROM golang:1.22.5
+
+WORKDIR /appdir
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY ./ ./
+
+RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -o /scheduler.bin
```

В линуксе обычно нет расширения для исполняемых файлов
```suggestion
RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -o /scheduler
```